### PR TITLE
styling of baseRate inputs

### PR DESCRIPTION
**Pull request by rkzel**

### CODE REVIEW

File: apps/projects/app/components/Content/Settings.js [hunk 1]
```diff
@@ -340,17 +328,23 @@ const BaseRate = ({ baseRate, onChange }) => (
       Define your organization’s hourly rate. This is multiplied by the bounty
       size and converted into the bounty currency under the hood.
     </Text.Block>
-    <Field label="RATE PER HOUR">
+    <FieldTitle style={{marginBottom: '0px'}}>Rate per hour</FieldTitle>
```
Comment (ottodevs) [inline apps/projects/app/components/Content/Settings.js]: For consistence and convention zero values are passed without units on styles

File: apps/projects/app/components/Content/Settings.js [hunk 2]
```diff
@@ -340,17 +328,23 @@ const BaseRate = ({ baseRate, onChange }) => (
...
+    <StyledInputDropDown style={{marginBottom: '0px'}}>
```
Comment (ottodevs) [inline apps/projects/app/components/Content/Settings.js]: same about zero values on styles

File: apps/projects/app/components/Content/Settings.js [hunk 3]
```diff
@@ -340,17 +328,23 @@ const BaseRate = ({ baseRate, onChange }) => (
...
       <NumberFormat
         customInput={StyledNumberInput}
         fixedDecimalScale
         decimalScale={2}
         value={baseRate}
         allowNegative={false}
-        onChange={onChange}
+        onChange={onChangeRate}
+        style={{marginRight: '0px'}}
```
Comment (ottodevs) [inline apps/projects/app/components/Content/Settings.js]: same about zero values

File: apps/projects/app/components/Content/Settings.js [hunk 4]
```diff
@@ -331,7 +299,27 @@ const BountyContractAddress = ({ bountyAllocator }) => (
   </div>
 )
 
-const BaseRate = ({ baseRate, onChange }) => (
+const StyledInputDropDown = styled.div`
```
Comment (ottodevs) [inline apps/projects/app/components/Content/Settings.js]: We should avoid duplicating code and use the existing InputDropDown component instead

File: apps/projects/app/components/Content/Settings.js (hunk 1 above)
Comment (rkzel) [reply apps/projects/app/components/Content/Settings.js]: A-ha. Sure.

File: apps/projects/app/components/Content/Settings.js (hunk 4 above)
Comment (rkzel) [reply apps/projects/app/components/Content/Settings.js]: I have tried that, but InputDropDown has predefined Text Input field, and in this case the field is NumericInput instead. Perhaps InputDropDown needs to be generalised to accept arbitrary fields, but it felt out of scope...

**Comment by ottodevs** [comment]
LGTM I basically agree Quazia's comment

**Pull request merged**